lms/services/lti_role_service.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+from sqlalchemy.orm import Session
+
+from lms.models import LTIRole
+
+
+class LTIRoleService:
+    """A service for dealing with LTIRole objects."""
+
+    def __init__(self, db_session: Session):
+        self._db = db_session
+
+    def get_roles(self, role_string) -> List[LTIRole]:
```

Comment:
`role_string` / `roles :str` or `lti_roles:str`? 

I'll probably would have written that but not a strong preference

Reply:
Yeah I'll have a think. Just trying to differentiate it from the `LTIRole` object, which it absolutely isn't, and to imply that it's singular, not a list.

The type annotation does this of course, but only here, rather than in the caller.